spec/requests/events_spec.rb

```diff
@@ -0,0 +1,81 @@
+require 'rails_helper'
+require 'support/request_helper'
+
+RSpec.describe "GET events/:id", type: :request do
+  describe "Availablities table shows correct availablities" do
+    before(:each) do
+      login_as_editor
+
+      @event_start_time = 2.hours.from_now
```

Comment:
I'm not a fan of this before each block.  It would be better to put this logic in the tests themselves, but there would need to be work done on the availabilities factories to make the code clean.  I was running out of time that I could put on this though.  I can come back to this at a later date.

Reply:
Is your concern around line 9 specifically ?

Reply:
no, the whole before block is a little brittle.  I think the next step for these tests are to make some more expressive factories, but I wanted to be clear on the intended behavior before moving forward on that.

Reply:
Thanks for your help.
If you get to the factories, I have a preference that they not create associated objects. I have seen that become a cascade of things being created that are not needed for the test at hand. Associated objects can be created in the spec itself.

Reply:
Agreed, that is a better practice.